fastq_filtrator.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+# MAIN
+
+input_fastq = eval(input("Введите аргумент input_fastq: "))
+output_file_prefix = eval(input("Введите аргумент output_file_prefix: "))
+gc_bounds = [i for i in input("Введите аргумент gc_bounds: ").split()]
+length_bounds = [i for i in input("Введите аргумент length_bounds: ").split()]
+quality_threshold = [input("Введите аргумент quality_threshold: ")]
+save_filtered = input("Введите аргумент save_filtered: ")
+prefix_filtered = "_passed.fastq"
+prefix_failure = "_failed.fastq"
+
+def gc_bounds1(up=100, lo=0, SEQ=[]):
+    if up > lo:
+        up_broad = up # верхняя граница
+        low_broad = lo # нижняя граница
+    else:
+        up_broad = lo # верхняя граница
+        low_broad = up # нижняя граница
+    numbers_of_reads = [] #
+    counter = 0
+    for i in SEQ[0::2]: # ЗДЕСЬ ИСХОДНЫЕ ДАННЫЕ УЖЕ ПЕРЕПИСАНЫ КАК k
+        counter+=1
+        num = (i.count("G") + i.count("C"))*100/len(i) # подсчет количества GC% для строки
+        if num < up_broad and num > low_broad: # условия фильтрации по границам
```

Comment:
Все интервалы в задании включают границы
```suggestion
        if low_broad <= num <= up_broad: # условия фильтрации по границам
```